rsfmodel/rsf.py

```diff
@@ -202,9 +258,36 @@ def _check_slider_displacement(self, tol=0.01):
             return True
 
     def _calculateContinuousDisplacement(self, velocity):
+        """
+        Calculate the displacement of the slider with cumtrapz.
```

Comment:
Instead of giving an implementation detail (the use of `cumtrapz`), maybe instead just say "by integrating the velocity"?